bindings/rust/Cargo.toml

```diff
@@ -7,15 +7,30 @@ links = "ckzg"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+
+# BLST Compilation:
+# By default, compile with ADX extension if the host supports it. Binary can be
+# executed on systems similar to the host.
 default = ["mainnet-spec"]
 mainnet-spec = []
 minimal-spec = []
+# BLST Compilation:
+# Compile in portable mode, without ISA extensions.
+# Binary can be executed on all systems.
+portable = []
+# BLST Compilation:
+# Enable ADX even if the host CPU doesn't support it.
+# Binary can be executed on Broadwell+ and Ryzen+ systems.
+force-adx = []
+# BLST Compilation:
+# Suppress multi-threading.
+# Engaged on wasm32 target architecture automatically.
+no-threads = []
```

Comment:
Nice. I see that, for the most part, this is [copied from blst](https://github.com/supranational/blst/blob/e9dfc5ee724b5b25d50a3b6226bee8c2c9d5e65d/bindings/rust/Cargo.toml#L26-L38).

Reply:
yeah very much